VersionHistory.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+### 2.2
```

Comment:
```suggestion
### 2.2.0
```

`Directory.Build.props` will need to be updated as well.

Reply:
Updated: https://github.com/FacilityApi/FacilityJavaScript/pull/26/commits/70887627d4db7840ad65fbbd03ee7b9e4f464c80